.travis/before_install-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ev
+
+sudo apt-get install -qq pkg-config fuse
+sudo modprobe fuse
+sudo chmod 666 /dev/fuse
+sudo chown root:$USER /etc/fuse.conf
```

Comment:
Will this make CI take much longer? or can travis reuse an image instead of installing FUSE every single time?